build/config/constants.js

```diff
@@ -39,7 +39,6 @@ const release_config = {
 
 const node_modules_paths = {
     binary_style: 'node_modules/@binary-com/binary-style',
-    deriv_p2p   : 'node_modules/@deriv/p2p',
```

Comment:
why this needs to removed?

Reply:
deriv_p2p was causing dependency issues and it is no longer used in the repo)